source/docs/articles/drupal/drupal-distributions-core-version-updates.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Drupal Distributions - Core Version Updates
+description: Learn to pull core updates for when repository maintainers are too slow.
+keywords: upstream, core, core update, core updates, upstream updates, manually pull updates, manually update, manually update upstream, pull upstream updates, pull core updates
+---
+Core updates for alternate distributions (Open Atrium, Commerce Kickstart, etc.) are serviced by the maintainer, but for the times in which those updates have not been pushed you can pull them into your site manually. For the security of your site, it is always best practice to run the most up to date version of Drupal core.
+
+## Communicate

```

Comment:
Re-order, place this paragraph last. Get users to their goal (updating core) as quickly as possible.